Bio/PDB/parse_pdb_header.py

```diff
@@ -140,7 +186,10 @@ def _parse_pdb_header_list(header):
         'structure_reference': "unknown",
         'journal_reference': "unknown",
         'author': "",
-        'compound': {'1': {'misc': ''}}, 'source': {'1': {'misc': ''}}}
+        'compound': {'1': {'misc': ''}}, 'source': {'1': {'misc': ''}},
+        'has_missing_residues': False,
+        'missing_residues' : []
```

Comment:
Please include a trailing comma here (ready for when someone adds more entries to the dict)